docs/technical-documentation/adding_format_jsonld.md

```diff
@@ -1,16 +1,27 @@
-By default, Islandora deploys with the `jsonld` module and `Milliner` microservice set to strip Drupal's Symfony-style
-`_format` query parameter.  This means that when your content is indexed in Fedora, the triplestore, etc... it's URI will
+Drupal requires the use of a `_format` query parameter to get alternate representations of a node/media.
+
+By default, Islandora deploys with the [jsonld](https://github.com/Islandora/jsonld) module and the [Milliner](https://github.com/Islandora/Crayfish/tree/master/Milliner) microservice. These two components are configured to strip this `_format` query parameter off of the end of URIs.
+
+This means that when your content is indexed in Fedora, the triplestore, etc... it's URI will
 be something like `http://localhost:8000/node/1` and not `http://localhost:8000/node/1?_format=jsonld`.
 
-If you are using a __very__ early version of Islandora 8 (pre-release), then you may have URIs with `_format=jsonld` at the
-end of them.  If you update to newer code, you will need to ensure that your site is configured to add `?_format=jsonld`
+## Pre-1.0 installations.
+
+If you are using a __very__ early version of Islandora 8 (pre-release), then you may have URIs with `_format=jsonld` at the end of them.
+
+If you update to newer code, you will need to ensure that your site is configured to add `?_format=jsonld`
 back to the URLs if you want to maintain consistency.
 
-- Go to `admin/config/search/jsonld` and confirm the 'Remove jsonld parameter from @ids' checkbox is unchecked.
-- Add `strip_format_jsonld: false` to `/var/www/html/Crayfish/Milliner/cfg/config.yaml`
+If you **don't** do this, you can end up with two copies of your objects in your Fedora repository (one with and one without `?_format=jsonld`). You will also have two sets of triples in your triplestore.
+
+### Adding ?_format=jsonld to your URIs
```

Comment:
Having the third-level heading looks odd since there is only one of each heading level. Consider making this a level-two header instead.